spec/support/domain_helpers.rb

```diff
@@ -47,6 +47,10 @@ def configure_dns(domain:, results:)
       .and_yield(resolver)
   end
 
+  def local_domain_uri
+    Addressable::URI.parse("//#{Rails.configuration.x.local_domain}").display_uri
```

Comment:
Why `display_uri` specifically? Are URIs we produce actually matching that?

Reply:
Grabbed that from another spot in app which was also doing a parse to grab the host ... but looks like it works fine as well on just the parse alone. Will update.